modules/ROOT/pages/index-appliance.adoc

```diff
@@ -0,0 +1,62 @@
+= Runtime Fabric Appliance
+
+
+== Runtime Fabric and Other PaaS Providers
+
+Anypoint Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
```

Comment:
suggest making the type of offering explicit -- Anypoint Runtime Fabric on VMs / bare metal

Reply:
Done. Is "Anypoint Runtime Fabric on VMs / bare metal" the formal name we're going with?